pages.nl/common/python.md

```diff
@@ -0,0 +1,36 @@
+# python
+
+> Python taal interpreter.
+> Meer informatie: <https://www.python.org>.
+
+- Start een REPL (interactieve shell):
+
+`python`
+
+- Voer een specifiek Python bestand uit:
+
+`python {{pad/naar/bestand.py}}`
+
+- Voer een specfiek Python bestand uit en start een REPL:
+
+`python -i {{pad/naar/bestand.py}}`
+
+- Voer een Python expressie uit:
+
+`python -c "{{expressie}}"`
+
+- Voer het script uit van een gespecificeerd bibliotheek module:
+
+`python -m {{module}} {{argumenten}}`
+
+- Installeer een pakket met `pip`:
+
+`python -m pip install {{pakket}}`
+
+- Debug interactief een Python script:
+
+`python -m {{pdb}} {{pad/naar/bestand.py}}`
+
+- Start de ingebouwde HTTP server op poort 8000 in de huidige map:
+
+`python -m {{http.server}}`
```

Comment:
```suggestion
`python -m pdb {{pad/naar/bestand.py}}`

- Start de ingebouwde HTTP server op poort 8000 in de huidige map:

`python -m http.server`
```

In these examples, the options are not really placeholders

Reply:
Agreed, the same must be updated on the English page too.